src/pages/tBTC/Bridge/index.tsx

```diff
@@ -46,24 +34,20 @@ const TBTCBridge: PageComponent = (props) => {
     )
   }, [dispatch, account])
 
+  const shouldRenderSidebars = pathname.startsWith("/tBTC/mint/deposit")
```

Comment:
What I would do here is I would create two separate layouts. One for `MintingProcess` and one for `Bridge` main page. They should have different URLs IMO:
- tbtc/bridge
- tbtc/bridge/mint

I think the main page for Bridge is not implemented in this PR so for now we could just create a MintLayout.

Reply:
It's only the example how we could conditionally render sidebars.
As I explained [here](https://github.com/threshold-network/bitcoin-on-base/pull/15#discussion_r1466426079) it's pointless to create separate layouts, just consume `PageLayout` and control it via props.